mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -73,7 +73,7 @@ class RegressionMetrics @Since("2.0.0") (
 
   /**
    * Returns the variance explained by regression.
-   * explainedVariance = $\sum_i (\hat{y_i} - \bar{y})^2 / n$
+   * explainedVariance = $\sum_i (\hat{y_i} - \bar{y})^2^ / n$
```

Comment:
Hm, I was going to say that this shouldn't be how you write exponents in latex, but, yeah it doesn't render correctly whereas `R^2^` below does. OK, if this fixes that issue, good.